content/blog/coredns-in-kubernetes-via-kubeadm.md

```diff
@@ -10,11 +10,33 @@ We can now install CoreDNS as the default service discovery via Kubeadm, which i
 
 Currently, [CoreDNS is Alpha in Kubernetes 1.9](https://github.com/kubernetes/features/issues/427). We have a roadmap which will make CoreDNS Beta in version 1.10 and eventually be the default DNS, replacing kube-dns.
 
+>It is important to note that currently when switching from kube-dns to CoreDNS, the configurations that come with kube-dns (stubzones, federations...) will no longer exist and will switch to a default configuration in CoreDNS.
+>The translation of the configurations from kube-dns shall be supported from the upcoming version of Kubernetes (v1.10) where CoreDNS will be Beta.
+
+
+## [Plugins](https://coredns.io/plugins/)
```

Comment:
I would propose to name the chapter "CoreDNS configuration"
add the stanza of the configuration we use for k8s.

explain that configuration of CoreDNS is made of plugin and keep the link to the plugins.